eng/pipelines/templates/jobs/integration.yml

```diff
@@ -49,3 +54,10 @@ jobs:
       arguments: >
         -BuildInfoPath '$(Pipeline.Workspace)/build_info/build_info.json'
         -NpmRegistry '${{ parameters.NpmRegistry }}'
+
+- template: /eng/pipelines/templates/jobs/update-mcp-repository.yml
```

Comment:
Does this mean we'll be publishing prerelease updates to the official registry pointing to internal/public package feeds?